### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kantord**
fixes https://github.com/kantord/emuto/issues/90

### CODE REVIEW

**Comment by kantord** [comment]
Woops, I have to add docs regarding project structure and I have to fix the commit message :-)

**Comment by kantord** [comment]
Ok, I pushed fixes :-)

**Comment by kantord** [comment]
Should be ready now, if tests don't fail

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Contribution guidelines
+
+First of all, thanks for contributing to emuto! All contribution is welcome.
+
+## Setup
+
+Make sure you have `node` and `yarn` installed.
+
+Fork the repo and clone your fork. Run `yarn checks` in the repo. It'll set up dependencies and makes sure everything works fine
```
[the remark this answers is not in the document]
Comment (jacekk) [reply CONTRIBUTING.md]:
```sh
$ yarn flow check
$ /home/jacekk/development/github/emuto/node_modules/.bin/flow check
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/graphviz/documentation.json:3:14

Unexpected token ILLEGAL

     1│ [ { tags: [],
     2│     description: 
     3│      { full: [32m'Module dependencies.',
     4│        summary: 'Module dependencies.',
     5│        body: '' },
     6│     ignore: false,
```

and if I replace the ignore section in `.flowconfig` with `<PROJECT_ROOT>/node_modules/.*/*.json`, I get 

```sh
$ yarn eslint src --fix
$ /home/jacekk/development/github/emuto/node_modules/.bin/eslint src --fix
Done in 1.76s.
sh: 1: jscpd: not found
```

:)

[the remark this answers is not in the document]
Comment (kantord) [reply CONTRIBUTING.md]: @jacekk do you still get the `Unexpected token ILLEGAL` error if you pull now?

[the remark this answers is not in the document]
Comment (kantord) [reply CONTRIBUTING.md]: I added jscpd as a dependency too :-)

**Comment by jacekk** [comment]
LGTM :+1:

**Comment by jacekk** [comment]
@kantord I guess the branch could be removed :)

**Pull request merged**